PATCHES

```diff
@@ -0,0 +1,17 @@
+diff --git a/src/c/include/recordio.h b/src/c/include/recordio.h
+index 4e1b78e..36d3803 100644
+--- a/src/c/include/recordio.h
++++ b/src/c/include/recordio.h
+@@ -73,6 +73,11 @@ void close_buffer_iarchive(struct iarchive **ia);
+ char *get_buffer(struct oarchive *);
+ int get_buffer_len(struct oarchive *);
+ 
++// airtime start
++#ifdef htonll  // Mac's arpa/inet.h defines this
++# undef htonll
++#endif  // htonll
++// airtime end
+ int64_t htonll(int64_t v);
+ 
+ #ifdef __cplusplus
+
```

Comment:
Move this to an airtime-specific directory?

Reply:
Moved